### PR TITLE
send mail when user register

### PR DESCRIPTION
**Pull request by namnn17b3**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/74057)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by namnn17b3** [comment]
ready

File: src/config/mail.connfig.ts
Comment (huongnt-2545) [inline src/config/mail.connfig.ts:1]: file name -> `mail.config.ts`

**Comment by namnn17b3** [comment]
ready

**Comment by namnn17b3** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**